### DISCUSSION TITLE
Error 500 in search method

### DISCUSSION THREAD

**Discussion by PilarMoralesViewnext** [OP-TED/eForms-SDK#710] [Q&A] [answered]
Hello,

We have found some response code 500 in the preview environment when the process that control the change of status in notice call ed the search method. 

Not always, but often. Is this a normal behaviour? Are we doing something wrong?

I copy the part of the log when the call works and it doesn't work, in case it helps:

_**2023-10-06T13:31:30+02:00 DGPEDP11 [PRE1_INTRA] [0x80e0015b][mpgw][info] mpgw(MPG_REST_DOUE_V2): trans(4074869855)[172.17.241.94] gtid(866c07f7651ff012f2e1945f): HTTP response code 500 for '[https://enotices2.preview.ted.europa.eu/esenders/api/v2/notice/notices/search-esentool?page=0&pageSize=20&sortProperty=updatedAt&sortOrder=DESC&statuses=STOPPED&statuses=PUBLISHED&statuses=NOT_PUBLISHED'](https://enotices2.preview.ted.europa.eu/esenders/api/v2/notice/notices/search-esentool?page=0&pageSize=20&sortProperty=updatedAt&sortOrder=DESC&statuses=STOPPED&statuses=PUBLISHED&statuses=NOT_PUBLISHED%27)**_

**_2023-10-06T13:46:31+02:00 DGPEDP11 [PRE1_INTRA] [0x80e0015b][mpgw][info] mpgw(MPG_REST_DOUE_V2): trans(4077333311)[172.17.241.94] gtid(866c07f7651ff396f3072b3f): HTTP response code 200 for '[https://enotices2.preview.ted.europa.eu/esenders/api/v2/notice/notices/search-esentool?page=0&pageSize=20&sortProperty=updatedAt&sortOrder=DESC&statuses=STOPPED&statuses=PUBLISHED&statuses=NOT_PUBLISHED'](https://enotices2.preview.ted.europa.eu/esenders/api/v2/notice/notices/search-esentool?page=0&pageSize=20&sortProperty=updatedAt&sortOrder=DESC&statuses=STOPPED&statuses=PUBLISHED&statuses=NOT_PUBLISHED%27)_**

In other hand, it helps us to improve our systems for failed.

Thanks in advanced,

Pilar Morales

**Comment by pcolast** [comment, marked as answer]
Dear Pilar,

Could you please try again and let us know if the issue persists? If so, please contact us through Helpdesk so that we may look into it further.

Thank you.
Have a nice day,
Stefan
